src/components/tile/tile.style.ts

```diff
@@ -66,6 +66,9 @@ const getHighlightVariant = (variant: TileProps["highlightVariant"]) => {
       return "var(--colorsSemanticCaution500)";
     case "info":
       return "var(--colorsSemanticInfo500)";
+    // TO-DO: Replace HEX with token when available
```

Comment:
question: is there a ticket for this? Might be worth including in this comment etc